StackExchange.Profiling/Timing.cs

```diff
@@ -81,7 +81,25 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public IList<Timing> Children
+        {
+            get
+            {
+                lock(_lockObject)
+                {
+                    return _children == null ? null : new List<Timing>(_children);
+                }
+            }
+            set { _children = value == null ? null : new List<Timing>(value); }
+        }
+
+        /// <summary>
+        /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
+        /// </summary>
+        public IList<Timing> AccessChildren()
```

Comment:
this method is for the remaining 1% of the cases where the caller wants to modify Timings list. I'm thinking to replace it with explicit methods RemoveTimings or something like this.